packages/services/api/src/modules/shared/module.graphql.ts

```diff
@@ -7,12 +7,22 @@ export default gql`
   scalar JSONSchemaObject
   scalar SafeInt
 
+  extend schema
+    @link(url: "https://specs.apollo.dev/link/v1.0")
+    @link(url: "https://specs.apollo.dev/federation/v2.0", import: ["@tag"])
+
+  directive @link(url: String!, import: [String!]) repeatable on SCHEMA
+
+  directive @tag(
+    name: String!
+  ) repeatable on FIELD_DEFINITION | INTERFACE | OBJECT | UNION | ARGUMENT_DEFINITION | SCALAR | ENUM | ENUM_VALUE | INPUT_OBJECT | INPUT_FIELD_DEFINITION | SCHEMA
```

Comment:
We turn our graphql schema into an apollo subgraph, so we can run our federation logic for filtering the schema on it :D